README.rst

```diff
@@ -1,19 +1,21 @@
 GeoPySpark
-***********
+**********
+
 .. image:: https://travis-ci.org/locationtech-labs/geopyspark.svg?branch=master
    :target: https://travis-ci.org/locationtech-labs/geopyspark
 
 .. image:: https://readthedocs.org/projects/geopyspark/badge/?version=latest
    :target: https://geopyspark.readthedocs.io/en/latest/?badge=latest
 
```

Comment:
Since we're moving down the `Contact and Support` section, it might make sense to add a Gitter badge here to make it easier for people to get there without having to scroll down the page.

Reply:
Good point and happy to make that change. Alternatively, should we move the `Contact and Support` section farther up? Perhaps Example -> Contact/Support -> Requirements + Setup instead? Or do both?

Reply:
I think having the `Contact and Support` section further down in the document makes sense. As it seems to interrupt the flow of introducing GPS and its setup if it's earlier in the document.